source/routing/raptor_api.cpp

```diff
@@ -1363,6 +1370,18 @@ void make_response(navitia::PbCreator& pb_creator,
         return;
     }
 
+    for (const auto stop_point_iter : *departures) {
+        SpIdx sp_idx = stop_point_iter.first;
+        navitia::type::StopPoint* stop_point = raptor.data.pt_data->stop_points[sp_idx.val];
+        LOG4CPLUS_TRACE(logger, "departure : " << stop_point->uri << " distance : " << stop_point_iter.second);
+    }
+
+    for (const auto stop_point_iter : *destinations) {
+        SpIdx sp_idx = stop_point_iter.first;
+        navitia::type::StopPoint* stop_point = raptor.data.pt_data->stop_points[sp_idx.val];
```

Comment:
```suggestion
        auto stop_point = raptor.data.pt_data->stop_points[sp_idx.val];
```

Reply:
Just for good practice :call_me_hand:

Reply:
Done 051ab1a4d1bf54e95d2a5d195910e797830ad317